nano/lib/stats.hpp

```diff
@@ -387,7 +387,11 @@ class stat final
 		generator_broadcasts,
 		generator_replies,
 		generator_replies_discarded,
-		generator_spacing
+		generator_spacing,
+		
+		// Block pipeline details
+		early_verify_filter_early,
```

Comment:
Should these be primary counters? If we add such detail at this level, this list will get very very long.

Reply:
Moving them to primary counters doesn't seem like it would reduce the count. We could do pass/reject as in/out for direction which would cut these in half. I think the nice part about having them under the same primary counter is that it could be filtered by that counter to get just the block pipeline stats.